tensorboard/plugins/projector/tf_projector_plugin/index.js

```diff
@@ -0,0 +1,20 @@
+export function render() {
```

Comment:
I'm a bit confused at the role of this rendering logic.  Since this is fully controlled by the plugin, this should be running in an iframe, right?  But then it's also creating its own iframe and inserting the projector HTML there?

Is that the typical flow that we expect for plugins based on custom web components - our iframe holds plugin code that creates its own iframe?  It seems a little convoluted.

Reply:
> Is that the typical flow that we expect for plugins based on custom web components

Yes but with a nuance: Yes you are using WebComponents that require HTML counterpart -- i.e., Polymer 1 and 2. Technically, we can still be using HTML import (which seems to work in my Chrome 75), I didn't want to rely on obsolete browser feature. Without using HTML import, a way to load HTML binary would be using an iframe which is the reason for this nested iframe.